### DISCUSSION TITLE
Добавить возможность после развертывания выключить автоматическое дополнение в auto.list

### DISCUSSION THREAD

**Discussion by elijah0107** [Anonym-tsk/tpws-keenetic#7] [Идеи и предложения]
Спустя день, два работы хочется выключить заполнение, чтобы работал только лист из тех что уже есть

**Comment by Liiion911** [comment]
@elijah0107 вы можете в вашем конфиг файле найти EXTRA_ARGS и убрать там --hostlist-auto=... а содержимое уже собранных доменов из авто добавить в ваш юзерлист

**Reply by elijah0107** [reply]
Спасибо!

**Comment by Anonym-tsk** [comment]
В конфиге `--hostlist-auto=` переименуйте в `--hostlist=` и ваш авто список станет постоянным
